csharp.test/TestLogicalTypeFactory.cs

```diff
@@ -173,6 +173,15 @@ public static void TestWriteNoColumnNorWriterOverride_Array()
             TestWriteNoColumnNorWriterOverride(ArrayValues, ArrayCustomValues);
         }
 
+        [Test]
```

Comment:
Suggest you remove this test. One already exist in `TestColumn.cs`.

In fact, https://github.com/G-Research/ParquetSharp/pull/185/files removes the two calls to `IsSupported`. We can just add them back in both places.